Pref feature

jainakshansh opened the pull request:
Fixes #[105]

## Description
Created the Settings Activity and made the fragment inside it
Created the Preference Screen UI
Created the Custom Sign Out Dialog UI
Created the Custom Sign Out Java class for implementation


## Screenshots

![layout-2018-05-08-115409](https://user-images.githubusercontent.com/15801046/39741989-89214d44-52b9-11e8-81d7-ab5af93bf663.png)

![layout-2018-05-08-000504](https://user-images.githubusercontent.com/15801046/39741998-905464ac-52b9-11e8-8f2e-d63fd2aa53c4.png)

sagarsrao commented:
Akshansh Jain:)We appreciate your hard work behind this. We would like to bring some mistakes that you have done.
1.Confliction of the files that you have pushed.
2. Many hardcoded strings in every XML files and the standard way is to take it from styling resources files.
3. Sign out (feature)  that you have implemented should be a dialog
4. Dark theme is missing in the PR and the screenshots that you have shared.
Due to the above reasons we are closing this PR
Please take this as a learning. Thanks for contribution

jainakshansh commented:
Hey @sagarsrao 
+ Conflict only relates to some of the testing libraries that I removed because the project wasn't building. I'd have added them again, so that wasn't an issue. 
+ The Sign Out feature is a dialog. This is just the UI view that is shown. I've implemented the Java classes too.
+ I've made sure to include all the strings in the strings.xml, so I don't know what you are talking about.
+ DarkTheme can be converted to white in a matter of seconds. Please tell me if that is needed.
